commercetools-models/src/main/java/io/sphere/sdk/orders/messages/OrderMessage.java

```diff
@@ -6,8 +6,8 @@
 
 @JsonTypeInfo(
         use = JsonTypeInfo.Id.NAME,
-        property = "type",
-        visible = true
+        property = "type"
+        //defaultImpl = SimpleOrderMessage.class
```

Comment:
Is this commented for debug purposes.

Reply:
It is my mistake. It should be uncommented. It is used to deserialise messages not known to the jvm sdk.